src/app/docs/key-info/key-info-docs.component.html

```diff
@@ -0,0 +1,164 @@
+      >
+        <p>
+          Don't use key info when data doesn't require user attention or doesn't drive user action. Use <a href="https://developer.blackbaud.com/skyux/components/definition-list">definition lists</a> instead.
+        </p>
+
+        <sky-docs-design-guideline-thumbnail
+          caption="Don't use key info for data that isn't important summary information."
+          captionType="danger"
+          imageSource="~/assets/img/guidelines/key-info/key-info-usage-5.png"
+        >
+        </sky-docs-design-guideline-thumbnail>
+
+        <sky-docs-design-guideline-thumbnail
+          caption="Do use definition lists for content that doesn't require user attention."
+          captionType="success"
+          imageSource="~/assets/img/guidelines/key-info/key-info-usage-6.png"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Anatomy"
+    >
+      <sky-docs-anatomy
+        imagePath="~/assets/img/guidelines/key-info/key-info-anatomy.png"
+      >
+        <sky-docs-anatomy-item>Value</sky-docs-anatomy-item>
+        <sky-docs-anatomy-item>Label</sky-docs-anatomy-item>
+      </sky-docs-anatomy>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Options"
+    >
+      <sky-docs-design-guideline
+        heading="Layout"
+      >
+        <p>
+          Use the default vertical layout with the label under the value in most situations.
+        </p>
+
+        <sky-docs-design-guideline-thumbnail
+          imageSource="~/assets/img/guidelines/key-info/vertical-layout.png"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+
+      <sky-docs-design-guideline
+      >
+        <p>
+          Use the horizontal layout with the label beside the value for scenarios where vertical space is at a premium. For example, use the horizontal layout to display the total number of items in a list.
+        </p>
+
+      <sky-docs-design-guideline-thumbnail
+        imageSource="~/assets/img/guidelines/key-info/horizontal-layout.png"
+      >
+      </sky-docs-design-guideline-thumbnail>
+        </sky-docs-design-guideline>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Behavior & States"
+    >
+      <sky-docs-design-guideline
+        heading="Link styling"
+      >
+        <p>
+          You can style the key info component as a link when you want to let users click the value to navigate to a different location.
+        </p>
+      </sky-docs-design-guideline>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Related information"
+    >
+      <sky-docs-design-guideline
+        heading="Components"
+      >
+        <ul>
+          <li><a href="https://developer.blackbaud.com/skyux/components/definition-list">Definition list</a></li>
+        </ul>
```

Comment:
You have extra closing and opening `ul` tags here, which is adding extra space between the bulleted items because it's creating two separate lists instead of a list with 2 items in it.